.prettierrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   singleQuote: true,
   printWidth: 100,
   trailingComma: 'all',
+   arrowParens: 'avoid',
```

Comment:
My personal pref is to have arrowParens as `always`. Reason being - less refactoring, and less diffs created if a function signature changes. What's the rationale for using `avoid` here?

Reply:
Thanks for mentioning this - this was part of the React Native upgrade for some reason. I'm with you though, I'd rather have parentheses always.